spec/datadog/tracing/distributed/trace_context_spec.rb

```diff
@@ -429,7 +436,7 @@
 
         context "{ 'key' => 'value=with=equals' }" do
           let(:tags) { 't.key:value:with:equals' }
-          it { is_expected.to eq('_dd.p.key' => 'value=with=equals') }
+          it { is_expected.to eq('_dd.p.key' => 'value:with:equals') }
         end
```

Comment:
I think the description for this test is now outdated -- it previously tested that we deserialized `value:with:equals` back to `value=with=equals`, but now we don't do that (and while the test has been corrected, the description and strings still mention the equals which is kinda confusing).

Reply:
Nice catch! I updated the test to reflect what it actually wants to test.